src/_settings/mappings/career.json

```diff
@@ -0,0 +1,133 @@
+{
+  "properties": {
+    "active": {
+      "type": "boolean"
+    },
+    "applicant_types": {
+      "properties": {
+        "announcement_number": {
+          "type": "string"
+        },
+        "application_type": {
+          "properties": {
```

Comment:
Can we remove the `properties` object and access the `active` boolean right off of `application_type`?